File: BOJ/cpp/1924.cpp
#include <iostream>
#define endl '\n'
using namespace std;

int main() {
	int x, y;
	int month[12] = { 31, 28, 31, 30, 31, 30, 31, 31, 30, 31, 30, 31 };
	string day[7] = { "SUN", "MON", "TUE", "WED", "THU", "FRI", "SAT" };

	cin >> x >> y;

	for (int i = 0; i < x - 1; i++) {
		y += month[i];
	}
	cout << day[y % 7];

	return 0;
}

File: BOJ/python/string/1924.py
X, Y = map(int, input().split())
Day = 0
day = [31, 28, 31, 30, 31, 30, 31, 31, 30, 31, 30, 31]
week = ["SUN", "MON","TUE", "WED", "THU", "FRI", "SAT"]
for i in range(X-1):
    Day = Day + day[i]
print(week[(Day + Y) % 7])
